geomesa-ui/src/main/ui/bower.json

```diff
@@ -8,12 +8,11 @@
     "angular-resource": "~1.2.13",
     "bootstrap": "~3.1.1",
     "d3": "~3.4.2",
-    "jquery": "~2.1.0",
     "font-awesome": "~4.0.3",
+    "jquery": "~2.1.0",
     "lodash": "~2.4.1",
     "moment": "~2.7.0",
     "moment-timezone": "~0.1.0",
-    "leaflet-dist": "~0.7.2",
-    "leaflet-providers": "~1.0.1"
+    "openlayers" : "https://github.com/openlayers/ol3/releases/download/v3.2.1/v3.2.1.zip"
```

Comment:
The open layers repo doesn't seem to include the built artifacts, only source code, however, they are included in the zip files that can be found with their releases (IE there are download links to the zip files here https://github.com/openlayers/ol3/releases). Do you know of a way to get at these zip files in the manner you were speaking of?